sensorprocess/lidarsensorprocess.go

```diff
@@ -7,6 +7,8 @@ import (
 	"math"
 	"time"
 
+	"go.viam.com/rdk/components/camera/replaypcd"
```

Comment:
from what I can tell, this introduces a dependency on rdk purely for that `ErrorEndOfDataset` export. Is there a way we can forego this completely? Can we just have identical errors instead of an exported variable? That's more maintenance work for now, but cleaner for the future.

Reply:
I'd like use to always think about not cross-depending too much on rdk, helps far future us. 

I realize in some cases in this repo it's not possible - spatial math, RImage, etc. So would be okay to merge if you feel like this is overkill.

Reply:
since we own the replay sensors we could hardcode this error on the cartographer side. I think thats reasonable since we could already accidentally break this check if the version of viam-server used is different from the rdk version being imported here.

Reply:
I think for now I will leave it as is, but maybe we can think more about how to handle rdk dependencies that might break us

Reply:
Cool! alright with me!